_templates/_search_enterprise.html.erb

```diff
@@ -1,16 +1,15 @@
+<div id="hc-search">
+  <input id="hc-search-input" type="text">
+  <button id="hc-search-btn">Search</button>
+</div>
+
+<div id="hc-search-modal">
+  <div id="hc-modal-content">
+    <span id="hc-modal-close">&times;</span>
+    <div id="hc-search-results"></div>
+  </div>
+</div>
+
 <script>
-  (function() {
-    var cx = '013769182564158614814:e-dp46b51vk';
-    var gcse = document.createElement('script');
-    gcse.type = 'text/javascript';
-    gcse.async = true;
-    gcse.src = (document.location.protocol == 'https:' ? 'https:' : 'http:') + '//cse.google.com/cse.js?cx=' + cx;
-    var s = document.getElementsByTagName('script')[0];
-    s.parentNode.insertBefore(gcse, s);
-  })();
+  hcSearchCategory("docs_cp");
 </script>
-<% if version == '3.0' or version == '3.1' or version == '3.2' %>
-<gcse:search defaultToRefinement="Enterprise <%= version %>"></gcse:search>
-<% else %>
-<gcse:search defaultToRefinement="Container Platform <%= version %>"></gcse:search>
```

Comment:
This line is critical to the current behavior of the search interface. By specifying a 'refinement' (which is defined in the Google Custom Search interface), we can limit result to particular versions of the docs. Does your search replacement also partition search results by version?